tekton-task-images/helm/ubi.repo

```diff
@@ -0,0 +1,13 @@
+[ubi-8-baseos]
+name = Red Hat Universal Base Image 8 Server (RPMs)
+baseurl= https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/baseos/os
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1
+
+[ubi-8-appstream]
+name = Red Hat Universal Base Image 8 App Stream  (RPMs)
+baseurl= https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/baseos/os
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1

```

Comment:
This file can be removed by switching to the `ubi-minimal` image

Reply:
Thanks Petter! all done 🚀